pkg/proxy/apis/config/types.go

```diff
@@ -188,6 +190,14 @@ const (
 	ProxyModeKernelspace ProxyMode = "kernelspace"
 )
 
+// LocalMode represents modes to detect local traffic from the node
+type LocalMode string
+
+// Currently supported modes for LocalMode
+const (
+	LocalModeClusterCIDR LocalMode = "cluster-cidr"
```

Comment:
Is it normal to use `-` in the mode string? I'm assuming this was inherited from the `--cluster-cidr` flag, wondering if `clustercidr` makes more sense?

Reply:
fair point. in line with convention, will change to "clustercidr". The other values will similarly be like "nodecidr" etc.

Reply:
Done.